frontend/packages/dev-console/integration-tests/support/constants/pageTitle.ts

```diff
@@ -38,8 +38,10 @@ export const pageTitle = {
   PipelineRuns: 'PipelineRuns',
   Eventing: 'Eventing',
   RedHatOpenShiftServerless: 'Red Hat OpenShift Serverless',
-  CreateKnativeServing: 'Create KnativeServing',
-  CreateKnativeEventing: 'Create KnativeEventing',
+  RedHatCodeReadyWorkspaces: 'Red Hat CodeReady Workspaces',
+  CreateKnativeServing: 'Create Knative Serving',
+  CreateKnativeEventing: 'Create Knative Eventing',
```

Comment:
Page title is unchanged as per the latest code 
![9710-pr](https://user-images.githubusercontent.com/61005404/131965290-05a4a1ad-eacc-4eb6-9473-634b343839ab.png)

Can we restore the earlier values

Reply:
verified it on cluster version : 4.9.0-0.nightly-2021-09-01-193941